lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { AuthError, ClientAuthError } from "msal-common";
+
+/**
+ * BrowserAuthErrorMessage class containing string constants used by error codes and messages.
+ */
+export const BrowserAuthErrorMessage = {
+    noWindowObjectError: {
+        code: "no_window_object",
+        desc: "No window object detected."
+    }
+};
+
+/**
+ * Browser library error class thrown by the MSAL.js library for SPAs
+ */
+export class BrowserAuthError extends ClientAuthError {
```

Comment:
Can you elaborate on the reasoning for needing another class extension here? Does this `noWindowObjectError` not fit in any of the other error classes? Will this file eventually get extended?

Reply:
Will change to AuthError

Reply:
Done.